influxdb/client.py

```diff
@@ -292,6 +293,8 @@ def write(self, data, params=None, expected_response_code=204,
         if protocol == 'json':
             data = make_lines(data, precision).encode('utf-8')
         elif protocol == 'line':
+            if isinstance(data, str):
+                data = list(data)
```

Comment:
This will not do what you expect, instead it'll split out your string into a list as follows:

```
In [4]: tester = "onetwothree"

In [5]: [tester]
Out[5]: ['onetwothree']

In [6]: list(tester)
Out[6]: ['o', 'n', 'e', 't', 'w', 'o', 't', 'h', 'r', 'e', 'e']
```
Please replace with `data = [data]` and that should suffice.